actors/evm/src/interpreter/instructions/memory.rs

```diff
@@ -493,4 +494,107 @@ mod tests {
             assert_eq!(m.state.stack.pop().unwrap(), U256::from(32));
         };
     }
+
+    macro_rules! check_mem {
```

Comment:
can't this be a function?

Reply:
yes but we lose location info on failure in asserts, hence a macro.

Reply:
Ah... Ew. Ok, fair enough.